slack_sdk/socket_mode/aiohttp/__init__.py

```diff
@@ -164,16 +164,19 @@ async def monitor_current_session(self) -> None:
                         )
                     break
                 try:
-                    # The logging here is for detailed trouble shooting of potential issues in this client.
-                    # If you don't see this log for a while, it means that
-                    # this receive_messages execution is no longer working for some reason.
-                    counter_for_logging = (counter_for_logging + 1) % logging_interval
-                    if counter_for_logging == 0:
-                        log_message = (
-                            f"{logging_interval} session verification executed after the previous same log"
-                            f" ({session_id})"
-                        )
-                        self.logger.debug(log_message)
+                    if self.trace_enabled and self.logger.level <= logging.DEBUG:
```

Comment:
This debug logging is mainly for this SDK maintainers. We can disable this for most developers. That's why I added trace_enabled here.